Projects/Server/Geometry/Point2D.cs

```diff
@@ -51,8 +51,6 @@ public Point2D(Point2D p) : this(p.X, p.Y)
     {
     }
 
-    public override string ToString() => $"({m_X}, {m_Y})";
```

Comment:
If we revert this, the benchmarks won't be slower I think?

Reply:
Yeah, the longer execution time on this was a suprise. I'm trying to be clever to avoid competing, independent implementations of getting a string representation of this object (`ToString` vs interpolation, `$"{this}"`). I'd like to do a bit more exploration to understand better why there is a drop in performance and see if there is a solution to having one source of truth but keep the performance.

Reply:
Yeah I completely agree. I think the drop is in creating an intermediate ref struct and calling from one ref struct to another. It's only 8-10ns which makes sense.

Unfortunately, I don't think unlining will make this any better. I am ok with a 10ns penalty to keep the implementation standard. It only becomes a problem if you are doing some crazy like high perf text writing for a web service or file output with thousands of values.

Reply:
Nice! Let's do that. I should update Serial too.